### DISCUSSION TITLE
# We want Mojo 🔥 0.7

### DISCUSSION THREAD

**Discussion by Eprahim-taha** [modular/modular#1641] [General]
I'm excited, you're excited, everyone is excited to see what's new in Mojo and the amazing achievements of the team modular , I heard from one of my friends that Mojo 0.7 will support references and lifetime ,This is a very big change in Mojo. It will change Mojo for the better. Finally, Mojo will become safe, I can't wait to get Mojo 0.7 🥺😘

**Comment by iamtimdavis** [comment]
Good things come to those who wait 💯 Know we're working hard on it.

In the meantime, checkout Modverse #18 - https://www.modular.com/newsletters/modverse-weekly-issue-18

Stay tuned!

**Reply by Eprahim-taha** [reply]
Thank you for your interest ❤️
